cba_settings_userconfig/cba_settings.sqf

```diff
@@ -109,9 +151,11 @@ force force ace_hearing_unconsciousnessVolume = 0.4;
 
 // ACE Interaction
 force force ace_interaction_disableNegativeRating = true;
+force force ace_interaction_enableGroupRenaming = false;
```

Comment:
```suggestion
force ace_interaction_enableGroupRenaming = true;
```
This is what allows a group leader to rename their group, this is something we could make use of for when we mess with groups mid mission. Only the group leader can do it, it's not abusable if some does mess with it then they will be easily spotted